minidump/src/minidump.rs

```diff
@@ -992,6 +985,21 @@ impl Module for MinidumpModule {
                     Some(CodeId::from_binary(&raw.build_id))
                 }
             }
+            None if self.os == Os::Windows => {
+                // Fall back to the timestamp + size-based debug-id for Windows.
+                // Some Module records from Windows have no codeview record, but
+                // the CodeId generated here is valid and can be looked up on
+                // the Microsoft symbol server.
+                // One example might be `wow64cpu.dll` with code-id `378BC3CDa000`.
+                // This can however lead to "false positive" code-ids for modules
+                // that have no timestamp, in which case the code-id looks extremely
+                // low-entropy. The same can happen though if they *do* have a
+                // codeview record.
```

Comment:
I’m a bit unsure about this tbh. It is true that this would result in almost certainly unusable IDs, however AFAIK, none of the other tools validate this?